module/manifest.json

```diff
@@ -1,14 +1,15 @@
 {
-    "changes": "* Upgrade Trigger.io platform to `v2.8.6`\n* Update Parse iOS SDK to `1.17.2`\n* Update module to use Bolts v1.10.0 which replaces usage of UIWebView with WKWebView",
-    "dependencies": {
-        "bolts": {
-            "minimum_version": "1.10.0",
-            "maximum_version": "1.10.0"
-        }
-    },
-    "description": "Parse.com configuration to enable push notifications.\n\nThe parse module uses the Parse.com SDK to allow for native push notifications.",
-    "min_platform_version": "v2.8.6",
-    "namespace": "parse",
-    "platform_version": "v2.8.6",
-    "version": "3.4.0"
-}
+	"changes": "* Update Parse Android SDK to 1.20.0",
+	"dependencies": {
+		"bolts": {
+			"maximum_version": "1.10.0",
+			"minimum_version": "1.10.0",
+			"module": "bolts"
+		}
+	},
+	"description": "Parse.com configuration to enable push notifications.\n\nThe parse module uses the Parse.com SDK to allow for native push notifications.",
+	"min_platform_version": "v2.8.6",
+	"namespace": "parse",
+	"platform_version": "v2.8.6",
+	"version": "3.5.8"
+}
```

Comment:
Could you update the PR to now change the indentation for the JSON file here? 'cause git churn and 'cause all other JSON files in this repo use 4 spaces, too

Reply:
File is auto-generated by Toolkit.
Will change 👍

Reply:
Force-pushed [this new commit](https://github.com/Staffbase/trigger.io-parse/pull/10/commits/b6f2e8e60beeacaef9692db72c1cbdc999a71b11) with the correct diff.